client_unix_test.go

```diff
@@ -2,12 +2,36 @@
 
 package containerd
 
+import (
+	"runtime"
+)
+
 const (
 	defaultRoot    = "/var/lib/containerd-test"
 	defaultAddress = "/run/containerd-test/containerd.sock"
-	testImage      = "docker.io/library/alpine:latest"
+)
+
+var (
+	testImage string
 )
 
 func platformTestSetup(client *Client) error {
 	return nil
 }
+
+func init() {
+	switch runtime.GOARCH {
+	case "386":
+		testImage = "docker.io/i386/alpine:latest"
+	case "amd64":
+		testImage = "docker.io/library/alpine:latest"
```

Comment:
Let's just put this as the `default` case. In the future I hope this image is a multi-arch manifests and could cover the other cases and any that are potentially missed.